chrome/content/zotero/xpcom/server_connector.js

```diff
@@ -594,6 +594,53 @@ Zotero.Server.Connector.Progress.prototype = {
 };
 
 /**
+ * Translates resources using import translators
+ * 	
+ * Returns:
+ * 	- Object[Item] an array of imported items
+ */
+ 
+Zotero.Server.Connector.ImportTranslate = function() {};
+Zotero.Server.Endpoints["/connector/importFile"] = Zotero.Server.Connector.ImportTranslate;
```

Comment:
Name should match the endpoint. Maybe just `/connector/import`/`Connector.Import`, since the GUI function is just "Import…".